src/tasks/package.js

```diff
@@ -74,6 +84,23 @@ module.exports = (config) => {
         'ci:lint': 'npm run lint && npm run security',
         'ci:test': 'npm run test -- --runInBand',
         'ci:coverage': 'npm run test:coverage -- --runInBand',
+        'webpack-defaults': 'webpack-defaults',
```

Comment:
`webpack-defaults: 'webpack-defaults'` => `defaults: 'webpack-defaults'` ( not important, so don't mind me here :) )

```
$ npm run defaults
```